### ISSUE TITLE
Can't build the app in iOS

### ISSUE THREAD

**Issue by lucas-sesti** [GiancarloCode/form_bloc#65]
I'm trying to use the package but, when i put in my pubspec.yaml the flutter_form_bloc: ^0.12.2 (the latest version until now), i got some error messages: /flutter/.pub-cache/hosted/pub.dartlang.org/keyboard_utils-1.2.0/ios/Classes/KeyboardUtilsPlugin.m:2:9: fatal error: 'keyboard_utils/keyboard_utils-Swift.h' file not found
#import <keyboard_utils/keyboard_utils-Swift.h>, and i can't build my app, how can i fixed??

**Comment by GiancarloCode** [comment]
@lucas-sesti Hello :), Maybe I can change the plugin, but they all had bugs in the latest version of flutter.
I do not currently have a Mac, 
If you can find a plugin that works and detects the keyboard in iOS, I can easily migrate it, I will take care of trying it on Android and the Web.

**Comment by lucas-sesti** [comment]
Alright, i created an issue in the keyboard_utils and he already answered, he said will fixed this error, and when he fixed, i create new pull request to fix in flutter_form_bloc

**Comment by GiancarloCode** [comment]
Awesome, thank you :)

**Comment by GiancarloCode** [comment]
Hello, can you please use the git version, I changed the keyboard package but i want to know if it work on iOS
```yaml
dependencies:
  flutter_form_bloc:
    git:
      url: git://github.com/GiancarloCode/form_bloc.git
      path: packages/flutter_form_bloc
```

**Comment by GiancarloCode** [comment]
Solved in `flutter_form_bloc: 0.13.0`.

**Comment by lucas-sesti** [comment]
Thank you!
